code/__DEFINES/colors.dm

```diff
@@ -8,15 +8,16 @@
 #define COLOR_DARKMODE_TEXT "#a4bad6"
 
 #define COLOR_WHITE "#FFFFFF"
-#define COLOR_VERY_LIGHT_GRAY "#EEEEEE"
```

Comment:
because the RPD set the colors as undercase "#eeeeee" while the mapped ones are in uppercase since they use these defines. This caused that the mapped pipes couldn't connect at all to the RPD ones. 
I tried to just change the RPD colors in the .js to be all uppercases, but they'll still get set to undercases, so i had to change them on the defines

Reply:
When comparing hex rgb values you should be doing lowertext() to both sides wherever the comparison is happening if it's possible to have the wrong case come in as input

Reply:
So I revert the color.dm change and put lowertext() to the checks?

Reply:
preferably yeah

Reply:
done, next commit